schemas/documentation_generation.py

```diff
@@ -64,3 +75,38 @@ class UpdateFileDocsRequest(BaseModel):
 class UpdateFileDocsResponse(BaseModel):
     message: str
     id: str
+
+# GET /repos
+
+class GetReposResponse(BaseModel):
+    repos: list[str]
+
+# GET /repos/{repo_id}
+class GetRepoResponseRepoModel(BaseModel):
+    graph: dict[str, list[str]] | None
+    root_doc: str
+    completion_status: str
+    version: str # commitId
+
+    @classmethod
+    def from_firestore_model(self, firestore_model: FirestoreRepoCreateModel) -> 'RepoModel':
+        # Convert the status field
+        completion_status = self.calculate_completion_status(firestore_model.status)
+
+        return self(
+            graph=firestore_model.graph,
+            root_doc=firestore_model.root_doc,
+            completion_status=completion_status,
+            version=firestore_model.version
+        )
+    
+    # calculate the completion status e.g '3/5' for a repo
+    def calculate_completion_status(status_dict: dict[str, DocsStatusEnum]) -> str:
```

Comment:
it might be worth it to return the whole status object here. That way the UI can choose how they want to display this data, instead of only being able to use/parse a specific string "{completed}/{uncompleted}".

Reply:
So should we just return the same model that is in the database?

Reply:
Yes. If we add more internal fields later, we can numb it down to a simpler object (like we do with most other responses).